sharedbuffers/mapped_struct.py

```diff
@@ -4586,14 +4700,14 @@ def getter(self, proxy_into = None, no_idmap = False):
         proxy_class = self.proxy_class
         index = self.index
         idmap = self.idmap if not no_idmap else None
-        buf = self.buf
+        buf = self.wr_buf
 
         if proxy_class is not None:
             proxy_class_new = functools.partial(proxy_class.__new__, proxy_class)
         else:
             proxy_class_new = None
 
-        @cython.locals(pos=int)
+        @cython.locals(pos = int)
```

Comment:
Based on PEP8, I think that the original expression was ok

Reply:
Oh, right. It auto-corrected when I was modifying some code. I guess it shouldn't matter (and it's also more consistent with the rest of the code).